### DISCUSSION TITLE
Installation is done and working but After Restart, stuck on FLSUN boot

### DISCUSSION THREAD

**Discussion by romanopanzano** [Guilouz/Klipper-Flsun-Speeder-Pad#175] [Q&A]
Dear Super Guilouz, I installed your incredible version on my speeder pad, everything is working with the klipperScreen and mainsail.

But when I shutdown my pad and restart it, it satays blocked on the FLSUN boot. I need to know what I’ve done wrong.

**Comment by SckyzO** [comment]
It was a new install ? Or did you do an update ?

**Reply by romanopanzano** [reply]
I tried to install the guilouz version on the speeder pad following his instructions from the wiki.

**Reply by SckyzO** [reply]
So, you have missed a step i think

**Reply by romanopanzano** [reply]
but what ? good question ! I followed every step of the installation.

I don’t know how it’s possible, because it works with klipper, with the 3dcprinter also and when I restart, I’ve got the flsun’s screen boot blocked

**Reply by mithrandir63** [reply]
Are you sure ??? you don't reboot at this.time??![Screenshot_2024-06-20-21-00-17-836_com.android.chrome-edit.jpg](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/62623243/b3abfc06-125f-489b-b3ec-5e77f3a2ecb8)

**Reply by mithrandir63** [reply]
What is  the last step you are launch before the reboot ??

**Reply by crop24** [reply]
hello, you have an access with the ssh? you probably forget to put your usb serial into the printer.cfg
https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/wiki/Get-USB-Serial-from-Motherboard

**Reply by romanopanzano** [reply]
![Capture d’écran 2024-06-22 à 21 00 04](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/54840033/74ba2a5c-7a32-408a-8ad6-e490df8a374e)

After INSTALL OFFICIAL BUILDS (1 instance) 


When it's done, enter this commands (one at a time):

cd ~ && ln -s ~/gcode_files/USB-Disk ~/printer_data/gcodes/USB-Disk
sudo reboot

When the pad reboot, it's stay stuck on the FLSUN boot screen

**Reply by romanopanzano** [reply]
if i didn't launch sudo reboot and add to the moonraker.conf the following lines and add the usb serial into the printer.cfg

it works well. But if I switch off and restart the pad, it's stay also stuck on the FLSUN boot screen

**Comment by romanopanzano** [comment]
Dear all, after many tries, I installed perfectly Klipper version Guilouz on my speeder pad.

I found why, it was at the end of Update dependencies when you have to save your SSID with the password, I had an issue on it.

Thanks a lot for the help !!!

And THANKS Guilouz !

**Comment by brdi** [comment]
Greatz. Congratz.Op 22 jun 2024 om 23:10 heeft romanopanzano ***@***.***> het volgende geschreven:﻿
Dear all, after many tries, I installed perfectly Klipper version Guilouz on my speeder pad.
I found why, it was at the end of Update dependencies when you have to save your SSID with the password, I had an issue on it.
Thanks a lot for the help !!!
And THANKS Guilouz !
